src/libraries/System.Runtime/tests/System.Runtime.CompilerServices.Unsafe.Tests/UnsafeTests.cs

```diff
@@ -14,7 +14,7 @@ public class UnsafeTests
         public static unsafe void ReadInt32()
         {
             int expected = 10;
-            void* address = Unsafe.AsPointer(ref expected);
+            void* address = Unsafe.AsPointer(ref expected); // Unsafe.AsPointer is safe since expected is on stack
```

Comment:
Can these be updated to `&value` instead? These tests are not purposely testing `Unsafe.AsPointer`.

Reply:
See the above